### PR TITLE
template: Use CardBody component on CarouselCard

### PR DESCRIPTION
**Pull request by danigm**
https://phabricator.endlessm.com/T31912

### CODE REVIEW

File: template-ui/src/components/CardBody.vue [hunk 1]
```diff
@@ -35,7 +35,13 @@ import { mapGetters } from 'vuex';
 import { StructuredTags } from '@/constants';
 
 export default {
-  props: ['node'],
+  props: {
+    node: Object,
+    titleLines: {
```
Comment (manuq) [inline template-ui/src/components/CardBody.vue:40]: Nit: this new prop is not used at all. And the number of lines is hardcoded to 3 in the CSS anyways. [There are ways to do use props from the CSS with var()](https://stackoverflow.com/a/52280182/1178541). But if the prop has no users it should be a `data:`.

File: template-ui/src/components/CardBody.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+@function card-height($title-lines, $margins) {
```
Comment (manuq) [inline template-ui/src/components/CardBody.scss:1]: This function is great! A few things:
- I wonder why margins can't be derived from the number of lines.
- Maybe is better to start a `@/utils.scss` or move it to our `'@/styles.scss` directly, this file is not a component.

**Review by manuq — APPROVED** [review]
Looks great. Left some nits, feel free to take them or not. But please add a description to the commit message, it helps reviewing and to remember why and how we do things.

File: template-ui/src/components/CardBody.vue (hunk 1 above)
Comment (manuq) [reply template-ui/src/components/CardBody.vue:40]: I see the prop used now, I was confused by the camelcase. Still it's hardcoded in the call to the CSS function, so not very useful.

Comment (danigm) [reply template-ui/src/components/CardBody.vue:40]: This is to limit the number of lines in the title, so it's used in the `v-clamp`. It's also used in the css, but I've not found a clean way to share component properties with the css. I'll take a look to try to improve this.

File: template-ui/src/components/CardBody.scss (hunk 2 above)
Comment (danigm) [reply template-ui/src/components/CardBody.scss:1]: You are right about the margin, that shouldn't change because we're using the same component.

**Review by manuq — APPROVED** [review]

**Pull request merged**